### PR TITLE
DellEmc: Platform 2.0 Api(chassis,fan,eeprom) for S6100 and Z9100

### PR DESCRIPTION
**Pull request by gengankarthik**
**- What I did**
     1. Fan, Eeprom and Chassis support added for platform2.0 API.

**- How I did it**
     1. Fan, Eeprom and Chassis related API’s (Platform 2.0) is supported in DellEmc S6100 and Z9100 platforms.
 
**- How to verify it**
     1. Tested with a python script to load Chassis class and then call Fan and Eeprom APIs accordingly

UT logs attached with more details.
[S6100_Log.txt](https://github.com/Azure/sonic-buildimage/files/3444212/S6100_Log.txt)
[Z9100_Log.txt](https://github.com/Azure/sonic-buildimage/files/3444213/Z9100_Log.txt)

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c [hunk 1]
```diff
@@ -1738,24 +1743,39 @@ static ssize_t show_psu(struct device *dev,
...
+        case 2:
+            reg = FAN_3_SERIAL_CODE;
+            break;
+        case 3:
+            reg = FAN_4_SERIAL_CODE;
+            break;
+        case 4:
+            reg = FAN_5_SERIAL_CODE;
+            break;
+		case 10:
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c]: Cases 10 and 11 indented too far, not aligned with others

File: platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c [hunk 2]
```diff
@@ -1980,7 +2021,7 @@ static const struct attribute_group *smf_groups[] = {
         &smf_vsen_group,
         &smf_curr_group,
         &smf_tcpu_group,
-	&smf_dell_group,
+		&smf_dell_group,
```
Comment (jleveque) [inline platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c]: Indented too far

Comment (gengankarthik) [reply platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c]: Corrected the alignment ...

File: platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c (hunk 1 above)
Comment (gengankarthik) [reply platform/broadcom/sonic-platform-modules-dell/common/dell_pmc.c]: Thanks for your valuable feedback ...
Corrected the alignment ...

**Pull request merged**